### PR TITLE
Feat: Backend Init

### PR DESCRIPTION
**Pull request by SilviaPabon**
Includes:
- Basic doc about database and codeowners
- Basic files and folders about api
- Some necessary dev and normal dependencies
- Basic database conection and script
- docker-compose
- I used yarn as package manager
- I created rome config as a formatter and linter but I have some doubts about tab space

### CODE REVIEW

File: .github/docs/database.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    ADMINS {
+        STRING CEDULA PK "UNIQUE NOT NULL"
+        STRING NOMBRE "NOT NULL"
+        STRING CORREO PK "NOT NULL"
+        STRING PASSWORD "NOT NULL"
+        STRING APARTAMENTO "NOT NULL"
+        NUMBER ROL FK "NOT NULL, REF ROL.ID"
+    }
+
+    USERS {
```
Comment (PedroChaparro) [inline .github/docs/database.md:16]: This name doesn´t match with the table name (Residents).

File: .github/docs/database.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        STRING CEDULA PK "UNIQUE NOT NULL"
+        STRING NOMBRE "NOT NULL"
+        STRING CORREO PK "NOT NULL"
+        STRING APARTAMENTO "NOT NULL"
+        NUMBER ROL FK "NOT NULL, REF ROL.ID"
+    }
+
+    ROL {
+        NUMBER ID PK "UNIQUE, NOT NULL"
+        STRING TIPO "NOT NULL"
```
Comment (PedroChaparro) [inline .github/docs/database.md:26]: I suggest replacing this field with "NAME" and adding a new field called "DESCRIPTION" to briefly describe each role.

File: database/init.psql [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+(
+    id SERIAL PRIMARY KEY UNIQUE,
+    tipo VARCHAR ( 1 ) NOT NULL
+);
+
+-- DROP TABLE IF EXISTS admins;
+
+CREATE TABLE IF NOT EXISTS admins
+(
+    cedula VARCHAR ( 12 ) PRIMARY KEY UNIQUE,
```
Comment (PedroChaparro) [inline database/init.psql:28]: I don´t think we need to limit the "cedula" length.

File: database/init.psql [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    name VARCHAR ( 150 ) NOT NULL,
+    mail VARCHAR ( 230 ) NOT NULL UNIQUE,
+    password VARCHAR ( 250 ) NOT NULL,
+    idRol SMALLINT NOT NULL,
```
Comment (PedroChaparro) [inline database/init.psql:32]: What do you think about using SCREAMING_SNAKE_CASE for all the tables and fields names?

Comment (SilviaPabon) [reply database/init.psql:32]: IM_AGREE_!

File: rome.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (PedroChaparro) [inline rome.json:1]: Seems good, lets see how good it is compared with prettier and eslint.

**Review by PedroChaparro — APPROVED** [review]
Let's start

**Pull request merged**